Fixture/FixtureGroupInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Doctrine\Bundle\MongoDBBundle\Fixture;
+
+/**
+ * FixtureGroupInterface can be implemented by fixtures that belong in groups
+ */
+interface FixtureGroupInterface
```

Comment:
I'm not excited to introduce a separate interface for this here, as it means that people that want to use their fixtures for ORM and ODM will have to use two different marker interfaces that provide the same functionality. However, since the corresponding ORM interface resides in the bundle and the library itself doesn't know anything about groups, there might not be a clean way around this 🤔 

I'll have to give this some more thought before merging this.

Reply:
The symfony  loaders are also seperate right now because you need the ODM/ORM compiler pass interface, so if you would want to combine the 2 we need to either depend on the fixtureBundle and extend their loader or abstract the loading from the type of database adatper you need and use injection for the ObjectManager in the load() function.

If the fixtures have proper dependency injection, injecting the objectManager in the constructor avoids needing to have the ODM/ORM distinction between the compiler pass marker and you can use just one loader with one interface. 

Abstracting this into a separate bundle makes more sense than including it in the mongodb bundle i guess. 

 I'm not sure how many people use both ORM and OMD anyway? In our project we only use the ODM, so we don't need the FixturesBundle and the assorted doctrine and doctrine cache bundles it takes with it. 

Maybe for 4.0 we could put the fixture stuff in a seperate bundle that depends or extends on the FixturesBundle and override the symfony loader from the fixturesbundle to use the same Fixture interfaces and use dependency injection for the ObjectManager so it can work with both the ORM and ODM stuff. In a newer FixtureBundle they could also move to the injected ObjectManager so you don't need it anymore in the load(). 
It would mean that people that migrate to 3.6 need to change things again for 4.0, but only if they want to combine ORM/ODM. 
It's all a bit messy...